pegasus/sites.v3/hourofcode.com/views/learn_carousels.haml

```diff
@@ -9,39 +9,28 @@
 
 - @variation = variation
 
-#all-tutorials{:style=>'visibility:hidden; overflow:hidden;'}
+%h2.centered
+  = hoc_s(:hoc_beyond_carousel_title)
```

Comment:
Can you add the class `.heading-xxl` here to make this larger? It looks like the class that was on it was removed so now this `h2` and the "Students" `h2` it's the same size as the `h3`'s:

<img width="980" alt="Screenshot 2023-11-27 at 1 15 05 PM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/8da6667c-e3aa-4f3d-bf73-f7d16300240e">

----

### Should look like this:
<img width="996" alt="Screenshot 2023-11-27 at 1 15 53 PM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/28568424-302f-438f-955c-a23fa5e1c056">